decimal64math.go

```diff
@@ -133,12 +136,18 @@ func (d Decimal64) Quo(e Decimal64) Decimal64 {
 		return zeroes[sign]
 	}
 
+	if e == Zero64 || e == NegZero64 {
```

Comment:
We need `(Decimal64).IsZero64() bool`. Raise an issue for later.

Reply:
#15 - Done